umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/distributed_format_util.clj

```diff
@@ -4,6 +4,12 @@
   (:require
    [clojure.string :as string]))
 
+(defn remove-emtpy-strings
```

Comment:
use `defn-` to define private functions.

Reply:
fixed.